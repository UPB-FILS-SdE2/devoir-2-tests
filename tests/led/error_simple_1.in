led blink 200 3 9 9
exit