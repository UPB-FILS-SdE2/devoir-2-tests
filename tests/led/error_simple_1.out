cmd: Invalid count value.
cmd: 